src/styles/variables.scss

```diff
@@ -126,7 +126,7 @@ $highlight-color-second: rgba(206, 206, 60, 0.4);
 $highlight-color-third: rgba(192, 192, 49, 0.4);
 $highlight-color-focus: rgba(156, 230, 255, 0.5);
 $top-bar-height: 40px;
-$group-list-width: 300px;
+$group-list-width: 280px;
```

Comment:
Hmmmm....locally, the menu is rendering at <200px wide for me (i.e. too narrow):

![image](https://user-images.githubusercontent.com/439947/59116236-1de14100-8919-11e9-8676-fbc50f2902a8.png)

Reply:
This is resolved. I've added the ability for a `Menu` consumer to specify a custom class for the dropdown content area and used that to set a minimum width for the groups menu.